tutorials/uv-metrics/requirements.txt

```diff
@@ -1,9 +1,2 @@
+# Get it done
```

Comment:
So uv/master is not compatible with this? I thought those changes were in now?

Reply:
whoops, sorry to leave this in. yes, it is now compatible